scripts/script.js

```diff
@@ -1,38 +1,40 @@
 const cocktailApp = {};
 cocktailApp.drinks = {};
 cocktailApp.searchTypes = {};
-cocktailApp.searchTypes.name = '/api/json/v1/1/search.php?s=';
-cocktailApp.searchTypes.ingredient = '/api/json/v1/1/filter.php?i=';
-cocktailApp.endpoint = 'https://www.thecocktaildb.com';
-cocktailApp.ingredientPath = '/api/json/v1/1/filter.php';
-cocktailApp.form = document.querySelector('form');
-cocktailApp.drinkList = document.querySelector('.drink__list');
-cocktailApp.userInput = document.getElementById('search-bar');
+cocktailApp.searchTypes.name = "/api/json/v1/1/search.php?s=";
```

Comment:
let me know if you want to stick with "" instead of ''.
I will modify my branch accordingly~ ^^

Reply:
I actually prefer single quotes in my JS. Not sure how those double quotes slipped in there. My consistency in which quotes I use is definitely something I need to work on. Thanks for catching that!